tests/data_files/constant/server5-fan.crt

```diff
@@ -0,0 +1,10 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
I think we should update `tests/data_files/Makefile` to add rules for the copied files (or change the path for moved files). Even if we don't update them after they expire, the rules are useful: they document how the file was generated.

Maybe we should copy/move these rules to a new makefile in the `constant` directory?

Reply:
> Maybe we should copy/move these rules to a new makefile in the constant directory?

No. I do not think those files should be update, so we do not need a update makefile. 

We might need a Readme(Maybe a makefile ) to record the commands. And the commands MUST generate files same with previous version.

Reply:
Actually I agree with @gilles-peskine-arm here. I'm currently changing some of these files to the der format, and the fact that there are outdated rules in the makefile (pointing to a wrong path) caught me off guard - I had to move the files manually.

Reply:
Filed an issue [here](https://github.com/Mbed-TLS/mbedtls/issues/7816).